### DISCUSSION TITLE
Mouse Capture Issue: Player Addition in 'World' Scene Affects 'pp_stack' Scene Visualization

### DISCUSSION THREAD

**Discussion by leoncgp** [MenacingMecha/godot-psx-style-demo#22] [Q&A]
Hey!

I downloaded the template and started tinkering with it. However, when I added my player to the 'World' scene, everything works fine, but the mouse capture in the 'pp_stack' scene is locked. What do I need to do to enable it?

**Comment by MenacingMecha** [comment]
Check the "mouse filter" and "handle input locally" settings, I remember fixing this for something before but can't remember what I did.
